doc-content/enterprise-only/vehicle-route-planning/download-ref-imp-proc.adoc

```diff
@@ -0,0 +1,33 @@
+[id='download-ref-imp-proc_{context}']
+
+= Download and build the deployment files
+
+You must download and prepare the deployment files before building and deploying the application.
+
+.Procedure
+. To check the current Java installation, enter the following command:
++
+[source,shell]
+----
+java -version
+----
+
+. If necessary, install OpenJDK {VRP-JDK-VERSION}.
```

Comment:
Do we need this step if we already required it in preinstalled chapter?

Reply:
Removed.